lrrbot/commands/live.py

```diff
@@ -41,7 +41,7 @@
 	old_channels = {channel["channel"]["name"] for channel in follows}
 	old_channels.add(config["channel"])
 
-	futures = [twitch.follow_channel(channel, token) for channel in channels.difference(old_channels)]
+	futures = [twitch.follow_channel(channel) for channel in channels.difference(old_channels)]
```

Comment:
True, but this seems neater than having to pass the token in, and practically speaking it's unlikely there's going to be many iterations of this loop (majority case is 0, occasionally 1, very rarely more than that) so I don't think this is a concern that outweighs the neater code.